bin/lms-notify

```diff
@@ -357,7 +357,7 @@ if ($debtors_file)
 	# @TODO: check 'messages' table and don't send notifies to often
 	my $dbq = $dbase->prepare("SELECT c.id, c.pin, c.lastname, c.name,
 			SUM(value) AS balance,
-			(SELECT contact FROM customercontacts WHERE customerid = c.id AND type = 8
+			(SELECT contact FROM customercontacts WHERE customerid = c.id AND ((type & 8 = 8 OR type & 16 = 16) AND type & 32 <> 32)
```

Comment:
Gdyby tak zastosować: type & 24 > 0 AND type & 32 <> 32 ?
O co dokładnie chodzi w tym warunku?